tests/integration/test-wheelhouses.sh

```diff
@@ -1,19 +1,8 @@
 set -e
 
-# Download some wheels from one location
-PY_VERS="2.7 3.3 3.4 3.5"
-for PYVER in ${PY_VERS}; do
-    wget "https://github.com/getnikola/wheelhouse/archive/v${PYVER}.zip"
-    unzip "v${PYVER}.zip"
-done
-
 # Download some other wheels that Nathaniel made
-#URL=https://vorpus.org/~njs/tmp/manylinux-test-wheels/original/
-#curl --silent $URL | grep 'a href' | grep whl | cut -d ' ' -f 8 | cut -d '=' -f 2 | cut -d '"' -f 2 | xargs -n1 -I '{}' wget --no-check-certificate -P wheelhouse-njs "$URL/{}"
-
-# Download some more wheels that Robert made
-URL=http://stanford.edu/~rmcgibbo/wheelhouse/
-curl --silent $URL | grep 'a href' | grep whl | cut -d ' ' -f 8 | cut -d '=' -f 2 | cut -d '"' -f 2 | xargs -n1 -I '{}' wget -P wheelhouse-rmcgibbo "$URL/{}" -nc
+URL=https://vorpus.org/~njs/tmp/manylinux-test-wheels/original/
```

Comment:
Why not include these wheels in the source repo instead? We already have a few under the tests directory.

Reply:
I wouldn't want to clutter up the source repo with various multi-MB wheels; large binary files are a bit awkward to handle in git. This test can also easily be expanded to many more wheels, e.g. I saw [this scikit-learn wheelhouse](https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/) linked from #86.

Reply:
One way not to clutter up the source repo with various multi-MB wheels would be to use `git-lfs`.
> Do you think it's worth re-creating some more tests with real-world wheels, or is it more a nice-to-have alongside the main test suite? I wanted to make sure that whatever's there is running effectively, but I don't know how important it is.

Well, I think that's the main question here. I don't know if it's important either... Were those tests added to cover a feature, bug, in the past, I can't say looking at the history...